packages/menu-surface/index.tsx

```diff
@@ -253,62 +250,60 @@ class MenuSurface extends React.Component<MenuSurfaceProps, MenuSurfaceState> {
       getWindowScroll: () => {
         return {x: window.pageXOffset, y: window.pageYOffset};
       },
-      setPosition: (position: Position) => {
+      setPosition: (position: Partial<MDCMenuDistance>) => {
```

Comment:
Previously, `Position` had properties with string type. but `MDCMenuDistance` has properties with number type. You might need to append `px` when setting style.

Reply:
Good catch! It was actually working this way before (as a number). I also verified that it is automatically added on to the value when rendered. Do you think I should still add `px`?

Reply:
It must be, but I couldn't fin any documentation on this.